src/main/java/me/confuser/banmanager/runnables/ExpiresSync.java

```diff
@@ -38,7 +38,7 @@ public void run() {
         PlayerBanData ban = bans.next();
         banRecordStorage.addRecord(ban, plugin.getPlayerStorage().getConsole(), "");
 
-        banStorage.removeBan(ban);
+        banStorage.unban(ban, ban.getActor());
```

Comment:
This will cause multiple records to be added into the database against a player's history. 

Just to confirm, this will also cause the same unban event to trigger across multiple servers (where users are using local db sync), is that the behaviour you're expecting?

Reply:
Well, I expect BM to trigger PlayerUnbanEvent as I have zero willing to manually check if the player is unbanned 'cause their tempban time's up.